### ISSUE TITLE
Question(s)

### ISSUE THREAD

**Issue by Fireworkstars46** [ish-/AuRo#3]
Is there a way to get this addon to work on all sites?

**Comment by ish-** [comment]
@Fireworkstars46  No. But the most, that doesnt use AudioContext. Its written in disclamer.
At the same time you could ask for specific site, it could improve app if it will be a bug

**Comment by AnnasSvnTYouth** [comment]
To use this extension on this domain click "Allow" this site to use your audio.
Also if a dialog doesn't appear, permission could be blocked explicitly with your browser settings.

How can i fixed this? All of the site setting have allowed for the youtube.com
